### PR TITLE
Vote phase system message

### PR DESCRIPTION
**Pull request by alicjakujawa**
## Description

This PR adds in system message for voting phase

**New stuff** ✨

* Add types and message descriptors for Voting phase
* Displayed voting system message

Screenshot 2021-04-22 at 15 49 26

**TODO** 
* Rebase with reputation-voting whenever #2451 will be merged 

Resolves DEV-289

### CODE REVIEW

File: src/data/resolvers/motions.ts [hunk 1]
```diff
@@ -198,6 +208,33 @@ export const motionsResolvers = ({
...
+        }
+      }
+
+      if (motionNetworkState === NetworkMotionState.Failed) {
+        systemMessages.push({
+          type: ActionsPageFeedType.SystemMessage,
+          name: SystemMessagesName.MotionHasFailedNotFinalizable,
+          createdAt: motion.events[1].toNumber() * 1000,
+        });
+      }
```
Comment (rdig) [inline src/data/resolvers/motions.ts:236]: Why is this here? I just reviewed this code #2451 

Was this cherry-picked ?

**Review by rdig — APPROVED** [review]
Good to go as well.

**Review by chinins — APPROVED** [review]
Great work!

File: src/data/resolvers/motions.ts (hunk 1 above)
Comment (alicjakujawa) [reply src/data/resolvers/motions.ts:236]: hehe, i create PR from that branch as i needed changes from it :) I added in todo to rebase it before merge so no worry
